etc/webpack/webpack.config.js

```diff
@@ -3,6 +3,7 @@ const webpack = require('webpack');
 const OptimizeCssAssetsPlugin = require('optimize-css-assets-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const StyleLintPlugin = require('stylelint-webpack-plugin');
+const { BundleAnalyzerPlugin } = require('webpack-bundle-analyzer');
```

Comment:
Let's put the bundle analyzer in a different PR. It's related to the _investigation_ of omitting the react libraries from the bundle, but technically a different (development) feature and irrelevant for users. The exclusion of the react libraries _is_ relevant for users.